.github/workflows/image-canary.yaml

```diff
@@ -0,0 +1,16 @@
+name: ImageCanary
+on:
+  workflow_dispatch: {}
+  schedule:
+  - cron: '0 */4 * * *'
+jobs:
+  ImageCanary:
+    runs-on: ubuntu-latest
+    steps:
+    - name: pull image
+      run: |
```

Comment:
How will we be notified if this canary fails?

Reply:
I can add a slack notifier, that's a good point